agent/lib/kontena/workers/fluentd_worker.rb

```diff
@@ -0,0 +1,60 @@
+require 'fluent-logger'
+
+module Kontena::Workers
+  class FluentdWorker
+    include Celluloid
+    include Celluloid::Notifications
+    include Kontena::Logging
+
+    attr_reader :fluentd, :node_name, :queue
+
+    ##
+    # @param [Queue] queue
+    # @param [Boolean] autostart
+    def initialize(autostart = true)
+      @fluentd = nil
+      @queue = []
+      @forwarding = false
+      info 'initialized'
+      subscribe('agent:node_info', :on_node_info)
+      subscribe(Kontena::Workers::ContainerLogWorker::EVENT_NAME, :on_log_event)
```

Comment:
50/60 :)
Maybe I'm just old-school, but I find using "random strings" a bit smelly in sense. We do it quite a lot in our code base... 
But I can live with plain strings if others see that as better option.